templates/ContentGenerator/Instructor/PGProblemEditor/revert_form.html.ep

```diff
@@ -28,8 +28,9 @@
 
 		<div class="row align-items-center mb-2">
 			<div class="col-auto">
+				% param('action.revert.type', 'backup') unless $foundTempFile;
```

Comment:
This will change the selected radio button from delete to restore if there is no temp file and you delete a backup. I think you also need to check that this parameter equals revert. The radio button backup also needs to be selected on page load if there is no temp file. I think following change of logic fixes this.

```
				% param('action.revert.type', 'backup')
					% if !$foundTempFile && (!param('action.revert.type') || param('action.revert.type') eq 'revert');
```

Reply:
Yeah, I guess that would be correct.